t/16-utils.t

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env perl -w
+
+# Copyright (C) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+BEGIN {
+    unshift @INC, 'lib';
+}
+
+use strict;
+use OpenQA::Utils;
+use Test::More;
+use Test::Mojo;
+use Test::Warnings;
+use Test::Output qw/stderr_like/;
```

Comment:
Thanks. Deleted unused uses. But I don't want to merge with `16-utils-runcmd.t` which loads the database and is therefore way slower than `16-utils.t` which is only using the functions which don't need the database and is near instantly finished.